lua/plugins/neo-tree.lua

```diff
@@ -27,6 +27,15 @@ return {
           state.commands.open(state)
         end
       end,
+      copy_file_name = function(state)
+        local node = state.tree:get_node();
+        vim.fn.setreg('*', node.name, 'c')
+      end,
+      copy_absolute_path = function(state)
+        local node = state.tree:get_node();
+        local full_path = node.path;
+        vim.fn.setreg('*', full_path, 'c')
+      end,
```

Comment:
This can be simplified to this, also AstroNvim uses the `+` register by default so that's probably a better default here. Also figured we could simplify the name

```suggestion
      copy_path = function(state) vim.fn.setreg("+", state.tree:get_node().path) end,
```